Jenkinsfile

```diff
@@ -0,0 +1,366 @@
+#!groovy
+
+/**
+ * This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which accompanies
+ * this distribution, and is available at https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+
+/*
+* zluxParameters is a map with key and value, if the value is empty it will automatically use staging for the key. 
+* To build a specific pull request just add the pull request number to the value of the repository you want to build. 
+*/
+
+def zluxParameters = [
+	"PR_ZLUX_APP_MANAGER":"",
+	"PR_ZLUX_APP_SERVER":"",
+	"PR_ZLUX_PLATFORM":"",
+	"PR_ZLUX_SERVER_FRAMEWORK":"",
+	"PR_ZLUX_SHARED":"",
+	"PR_ZLUX_BUILD":""
```

Comment:
considering its a brand new file, best to start off with consistent spacing. I see 4 here and 2 most elsewhere